### DISCUSSION TITLE
Support piping from `glob` to `ls` or allow `glob` results as argument of `ls`

### DISCUSSION THREAD

**Discussion by morzel85** [nushell/nushell#8684] [Ideas]
Nushell has a `glob` command with nice query options and useful arguments, it might even soon support [ignoring](https://github.com/nushell/nushell/issues/8537) directories (yeah, _node_modules_)!

It would be useful to be able to run something like `glob **/* -d 2 | ls -l` or `ls (glob **/* -d 2) -l)` to get a list of file/dir attributes with all the useful things that `glob` provides.

**Comment by fdncred** [comment]
You can do this 
```
glob **/* -d 2 | each {|r| ls -l -D $r} | flatten
```
